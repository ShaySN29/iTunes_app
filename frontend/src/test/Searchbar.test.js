import React from "react";
import renderer from "react-test-renderer";
import SearchBar from '../components/Searchbar.js'; // Importing component to test render


it('renders correctly', () => {
    const tree = renderer.create(<SearchBar />)
    .toJSON();
    expect(tree).toMatchSnapshot();
}); 


