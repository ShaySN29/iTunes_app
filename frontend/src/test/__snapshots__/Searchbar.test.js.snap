// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div>
  <div
    className="navBar"
  >
    <nav
      className="mb-4 navbar navbar-expand-lg navbar-light bg-success"
    >
      <div
        className="container"
      >
        <form
          className="d-flex"
        >
          <input
            aria-label="Search"
            className="me-3 form-control"
            onChange={[Function]}
            placeholder="Search"
            type="search"
          />
          <select
            aria-label="Default select example"
            className="ms-2 form-select"
            onChange={[Function]}
          >
            <option
              value="all"
            >
              All
            </option>
            <option
              value="music"
            >
              Music
            </option>
            <option
              value="podcast"
            >
              Podcasts
            </option>
            <option
              value="movie"
            >
              Movies
            </option>
            <option
              value="tvShow"
            >
              TV Shows
            </option>
            <option
              value="shortFilm"
            >
              Short Film
            </option>
            <option
              value="audiobook"
            >
              Audiobook
            </option>
            <option
              value="ebook"
            >
              Ebooks
            </option>
          </select>
          <button
            className="ms-2 btn btn-outline-light"
            disabled={false}
            onClick={[Function]}
            type="button"
          >
            Search
          </button>
        </form>
      </div>
    </nav>
  </div>
  <div
    className="searchResults"
  />
</div>
`;
